index.html

```diff
@@ -3794,7 +3794,7 @@ <h3>Ruby text annotations</h3>
 	<p class="advisement">Ruby implementations should allow annotations to appear on either or both sides of the base text.</p>
 	</div>
 	<div class="req" id="ruby_cjk">
-	<p class="advisement">Ruby markup in HTML is designed specifically for Chinese, Japanese, Korean, and Mongolian requirements, and should not be used as a general glossing mechanism.</p>
+	<p class="advisement">Ruby markup in HTML is designed primarily for Chinese, <a data-cite="jlreq#usage_of_ruby">Japanese</a>, Korean, and Mongolian requirements, and the needs of those languages should be prioritized in any use of the Ruby elements in other specifications. Within that constraint, Ruby may be used to associate annotations with a base string of text.</p>
```

Comment:
The I18N WG discussed this in our [2024-10-31 teleconference](https://www.w3.org/2024/10/31-i18n-minutes.html#dcd9). We don't quite agree with the changes proposed here. This is not because other annotation uses for Ruby do not exist in the wild, but that they are rare, usually can be better handled using other markup mechanisms, and because novel uses of Ruby conflict with their intended use. In addition, bear in mind that this document is guidance to specification authors and we seek to guide the audience the best we can.

Perhaps consider a modification like the following?

```suggestion
       <p class="advisement">Ruby markup in HTML is designed primarily for Chinese, Japanese, Korean, and Mongolian requirements. Annotation in other languages SHOULD use other mechanisms. Specifications SHOULD avoid proposing new uses for Ruby markup.</p>
      <p class="note">For examples of other types of annotation markup available in HTML, see <a href="https://r12a.github.io/blog/201708.html#20190304">post on Richard Ishida's blog </a>.</p>
```

Reply:
@r12a @xfq for visibility